packages/teleport/src/lib/tdp/client.ts

```diff
@@ -581,8 +582,8 @@ export default class Client extends EventEmitterWebAuthnSender {
   // so don't call this if your calling object is relying on listeners.
   // It's safe to call this multiple times, calls subsequent to the first call
   // will simply do nothing.
-  nuke() {
+  nuke(statusCode = StatusCodeEnum.NORMAL) {
```

Comment:
I've always found `nuke` to be a poor/insensitive choice of name here. Would `close` or `shutdown` be more appropriate?